tests/integration/ethernet_mtu_test.py

```diff
@@ -102,3 +102,17 @@ def test_decrease_to_ipv6_min_ethernet_frame_size_iface_mtu():
     netapplier.apply(copy.deepcopy(desired_state))
 
     assertlib.assert_state(desired_state)
+
+
+def test_decrease_to_lower_than_min_ipv6_iface_mtu():
+    desired_state = statelib.show_only(('eth1',))
+    origin_desired_state = copy.deepcopy(desired_state)
```

Comment:
IMHO `original` makes more sense than `origin` here. Also if you are touching this, it would seem to be more logical to me to first assign to `original_state` and then create a `desired_state` with a deep copy from it.